Resources/frontend/js/jquery.adyen-confirm-order.js

```diff
@@ -276,13 +276,26 @@
 
             var adyenConfig = me.getAdyenConfigSession();
 
-            me.adyenConfiguration = {
-                locale: adyenConfig.locale,
-                environment: adyenConfig.environment,
-                originKey: adyenConfig.originKey,
-                paymentMethodsResponse: adyenConfig.paymentMethodsResponse,
-                onAdditionalDetails: $.proxy(me.handleOnAdditionalDetails, me),
-            };
+            if (adyenConfig) {
+                //Get the config data from the session
+                me.adyenConfiguration = {
+                    locale: adyenConfig.locale,
+                    environment: adyenConfig.environment,
+                    originKey: adyenConfig.originKey,
+                    paymentMethodsResponse: adyenConfig.paymentMethodsResponse,
+                    onAdditionalDetails: $.proxy(me.handleOnAdditionalDetails, me)
```

Comment:
I think it should be the same but please double check it :)
```suggestion
                    onAdditionalDetails: me.handleOnAdditionalDetails.bind(me)
```

Reply:
Indeed it works, I just changed it. But I see that `handleOnAdditionalDetails` is not used anyway, but rather the details are checked here https://github.com/Adyen/adyen-shopware5/blob/develop/Resources/frontend/js/jquery.adyen-confirm-order.js#L106-L127